uaclient/entitlements/tests/test_fips.py

```diff
@@ -60,36 +60,25 @@ def entitlement(tmpdir):
 
 class TestFIPSEntitlementCanEnable:
 
-    def test_can_enable_true_on_entitlement_inactive(self, tmpdir):
+    @mock.patch(M_PATH + 'util.is_container', return_value=False)
+    def test_can_enable_true_on_entitlement_inactive(self, _mock, entitlement):
         """When operational status is INACTIVE, can_enable returns True."""
-        cfg = config.UAConfig(cfg={'data_dir': tmpdir.strpath})
-        cfg.write_cache('machine-token', dict(FIPS_MACHINE_TOKEN))
-        cfg.write_cache('machine-access-fips',
-                        dict(FIPS_RESOURCE_ENTITLED))
-        entitlement = FIPSEntitlement(cfg)
-        # Unset static affordance container check
-        entitlement.static_affordances = ()
         with mock.patch('sys.stdout', new_callable=StringIO) as m_stdout:
             assert True is entitlement.can_enable()
         assert '' == m_stdout.getvalue()
 
 
 class TestFIPSEntitlementEnable:
 
-    def test_enable_configures_apt_sources_and_auth_files(self, tmpdir):
+    @mock.patch(M_PATH + 'util.is_container', return_value=False)
```

Comment:
I don't think we need this mock here since the test mocks can_enable below inline (which is what called is_container)